lib/jekyll/filters.rb

```diff
@@ -357,6 +370,18 @@ def sort_input(input, property, order)
         .map!(&:last)
     end
 
+    private
```

Comment:
This really hurts my eyes 🙈 but it seems from the discussion in https://github.com/jekyll/jekyll/pull/6768 that it has to be like that. 😶